src/test/java/org/springframework/data/hazelcast/repository/query/HazelcastPropertyComparatorTest.java

```diff
@@ -0,0 +1,92 @@
+package org.springframework.data.hazelcast.repository.query;
+
+import org.junit.Test;
+
+import java.util.AbstractMap;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.assertEquals;
+
+public class HazelcastPropertyComparatorTest {
+
+    static class TestData {
```

Comment:
1. Could you make private?
2. Nit: I'd move the inner class to the bottom, because it's not the most important thing.

Reply:
1. & 2. Sure

Reply:
@leszko since we have `org.springframework.data.util.Pair` in the class path, what if we remove the TestData class altogether and use the former instead ?

Reply:
Found out that I can't do it since Pair doesn't accept null value